src/nrc/tests/commands/test_setup_configuration.py

```diff
@@ -115,24 +107,14 @@ def test_setup_configuration(self, m):
 
             self.assertEqual(response.status_code, status.HTTP_200_OK)
 
-    @requests_mock.Mocker()
-    def test_setup_configuration_selftest_fails(self, m):
-        m.get("http://open-notificaties.example.com/", exc=requests.ConnectionError)
-        m.get("http://open-notificaties.example.com/api/v1/kanaal", json=[])
-        m.get("https://oz.example.com/autorisaties/api/v1/applicaties", json=[])
-
-        with self.assertRaisesMessage(
-            CommandError,
-            "Could not access home page at 'http://open-notificaties.example.com/'",
-        ):
-            call_command("setup_configuration")
-
-    @requests_mock.Mocker()
-    def test_setup_configuration_without_selftest(self, m):
-        stdout = StringIO()
-
-        call_command("setup_configuration", no_selftest=True, stdout=stdout)
-        command_output = stdout.getvalue()
-
-        self.assertEqual(len(m.request_history), 0)
-        self.assertTrue("Selftest is skipped" in command_output)
+        # with self.subTest("Notifications configured correctly"):
+        #     config = NotificationsConfig.get_solo()
+        #     self.assertEqual(config.notifications_api_service, None)
+        #     self.assertEqual(config.notification_delivery_max_retries, 5)
+        #     self.assertEqual(config.notification_delivery_retry_backoff, 3)
+        #     self.assertEqual(config.notification_delivery_retry_backoff_max, 30)
```

Comment:
Similar to the comment [in the test file](https://github.com/open-zaak/open-notificaties/pull/201/files#diff-f6e6534d51058875815a05f879636b3241e9a772af170cb0b0f2043e439ea02d)